### DISCUSSION TITLE
Templates of reports

### DISCUSSION THREAD

**Discussion by xeno27** [inventree/InvenTree#5164] [Q&A] [answered]
Hi,

Iam a bit confused. I add via PR the possibility of creating a StockLocation Report. Therefor i add a Template for Stocklocation report.
This is located -> report/templates/inventree_slr_report.html . 
now comes the question:
how do i have to deal with the demo data, since all other templates are located in a  different folder (media/report/...) ?
What do i have to do to tell the server to copy this data automatically to the media/report ? 

I just want to integrate this in the demo data that everything works out of the box.

the Inventree PR: https://github.com/inventree/InvenTree/pull/5134
demo Data PR (i think this is wrong ) https://github.com/inventree/demo-dataset/pull/50

**Comment by SchrodingersGat** [comment, marked as answer]
@xeno27 refer to `report/apps.py` - when the server starts up, the sample reports are copied across

**Comment by xeno27** [comment]
Ok so i will create a function for the report stock location too :)

**Reply by SchrodingersGat** [reply]
Yep and you can close the PR for the demo dataset too

**Reply by xeno27** [reply]
i will :)
